tests/Persisters/Serializers/NativeTest.php

```diff
@@ -42,19 +46,45 @@ public function build() : void
         $this->assertInstanceOf(Native::class, $this->serializer);
         $this->assertInstanceOf(Serializer::class, $this->serializer);
     }
-    
+
     /**
      * @test
      */
     public function serializeUnserialize() : void
     {
         $data = $this->serializer->serialize($this->persistable);
-        
+
         $this->assertIsString($data);
 
         $persistable = $this->serializer->unserialize($data);
 
         $this->assertInstanceOf(DummyClassifier::class, $persistable);
         $this->assertInstanceOf(Persistable::class, $persistable);
     }
+
+    /**
+     * @return array<mixed>
```

Comment:
lol made changes before I could drop the last comment ... this works, `array[]` perhaps could be more specific ... not a big deal

Reply:
phpstan inspired me to this :shrug: :speak_no_evil:

Reply:
I love Stan. Check our contributing guide if you haven't already ... we've got all the custom composer commands listed, we try to use static analysis as much as possible.

https://github.com/RubixML/RubixML/blob/master/CONTRIBUTING.md